publiccode.yml

```diff
@@ -21,113 +15,67 @@ outputTypes:
 platforms:
   - GNU/Linux
```

Comment:
Is this right? Seeing the standard, this key should represent: `Questa chiave specifica su quale piattaforma funziona il software. È pensata per descrivere le piattaforme che l’utente userà per accedere ed utilizzare il software, piuttosto che la piattaforma sul quale il software gira.`